apps/test/unit/StudioAppTest.js

```diff
@@ -127,5 +127,35 @@ describe("StudioApp", () => {
           item.text === 'footer.how_it_works');
       expect(howItWorksItem.link).to.equal('https://studio.code.org/projects/gamelab/C_2x38fH_jElONWxTLrCHw/edit');
     });
+
+    it("returns How-It-Works item before Report-Abuse item in GameLab", () => {
+      project.getUrl.returns('https://studio.code.org/projects/gamelab/C_2x38fH_jElONWxTLrCHw');
+      var footItems = makeFooterMenuItems();
+      var howItWorksIndex = footItems.findIndex(item => item.text === 'footer.how_it_works');
+      var reportAbuseIndex = footItems.findIndex(item => item.text === 'footer.report_abuse');
+      expect(howItWorksIndex).to.be.below(reportAbuseIndex);
+    });
+
+    it("returns How-It-Works item before Report-Abuse item in AppLab", () => {
+      project.getUrl.returns('https://studio.code.org/projects/applab/l1RTgTXtyo9aUeJF2ZUGmQ');
+      var footItems = makeFooterMenuItems();
+      var howItWorksIndex = footItems.findIndex(item => item.text === 'footer.how_it_works');
+      var reportAbuseIndex = footItems.findIndex(item => item.text === 'footer.report_abuse');
+      expect(howItWorksIndex).to.be.below(reportAbuseIndex);
+    });
+
+    it("does not return Try-HOC menu item in GameLab", () => {
+      project.getUrl.returns('https://studio.code.org/projects/gamelab/C_2x38fH_jElONWxTLrCHw/');
+      var footItems = makeFooterMenuItems();
+      var tryHOCIndex = footItems.findIndex(item => item.text === 'footer.try_hour_of_code');
+      expect(tryHOCIndex).to.be.below(0);
```

Comment:
This might read a little nicer with a `not.to.include` assertion:
```javascript
const footItems = makeFooterMenuItems();
const itemTexts = footItems.map(item => item.text);
expect(itemTexts).not.to.include('footer.try_hour_of_code');
```